### ISSUE TITLE
`ctrl+w` to close active tabs

### ISSUE THREAD

**Issue by matekdev** [Facepunch/sbox-issues#5927]
### For?

S&Box

### What can't you do?

I want to be able to close tabs using `ctrl+w`
![image](https://github.com/user-attachments/assets/686a4b10-508c-431e-ba1f-94599e60da13)

### How would you like it to work?

Pressing `ctrl+w` closes the tab

### What have you tried?

Pressing `ctrl+w`

**Comment by garrynewman** [comment]
Wouldn't Ctrl+W conflict with movement?

**Comment by matekdev** [comment]
> Wouldn't Ctrl+W conflict with movement?

I might need to test tomorrow morning... but isn't this only a concern when right click is held down to activate the camera? Wouldn't ctrl + S run into the same problem (this binding already exists)?

**Comment by matekdev** [comment]
Yeah, it does not conflict with movement since right click isn't being held down.

It could potentially conflict with the W, E, R keybinds used for swapping between the gizmos (move, rotation, scale). Although I imagine you don't register the input if ctrl is being held down.
